From 9fe31c3fd47a41b1abb8c406459b7a81d4463b82 Mon Sep 17 00:00:00 2001
From: Leah Rowe <leah@libreboot.org>
Date: Tue, 29 Sep 2026 21:13:47 +0100
Subject: [PATCH 1/1] add fake makefile commands for lbmk integration

e.g. make ./mk -s not report errors. this is for ./mk test

Signed-off-by: Leah Rowe <leah@libreboot.org>
---
 Makefile | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7e604af..c11774f 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,18 @@
+all: int
 int: int.c
 	cc int.c -o int -O2 -lx86
 
-.PHONY: clean
+.PHONY: clean distclean oldconfig menuconfig savedefconfig nconfig
+distclean: clean
 clean:
 	rm -f int
+oldconfig:
+	:
+menuconfig:
+	:
+savedefconfig:
+	:
+olddefconfig:
+	:
+nconfig:
+	:
-- 
2.47.3

